examples/frame/src/frame/index.tsx

```diff
@@ -0,0 +1,62 @@
+// Copyright 2025 The Lynx Authors. All rights reserved.
+// Licensed under the Apache License Version 2.0 that can be found in the
+// LICENSE file in the root directory of this source tree.
+
+import { root, useState } from "@lynx-js/react";
+import type { FrameLoadEvent } from "@lynx-js/types";
+
+const App = () => {
+  const [bindLoadEvent, setBindLoadEvent] = useState({
+    url: "",
+    statusCode: 0,
+    statusMessage: "",
+  });
+  const [initData, setInitData] = useState<Record<string, unknown>>({
```

Comment:
Should we use `InitData` as type here?
```suggestion
  const [initData, setInitData] = useState<InitData>({
```

Reply:
In `<frame>`, the `data` prop is declared as `Record<string, unknown>`, and there doesn’t seem to be a suitable way to convert the `InitData` interface to `Record<string, unknown>`.